myprofile.php

```diff
@@ -90,7 +92,7 @@
         $error_array = array();
 
         if (isset($_POST['userID']) || isset($_GET['userID']) || $userID == "") {
-            die($_language->module['not_logged_in']);
+            die (generateAlert($_language->module['not_logged_in'], 'alert-danger'));
         }
```

Comment:
Kann man Zeile 94 bis 96 nicht sogar ganz raus nehmen? man hat doch schon eine if else abfrage in Zeile 30 bis 34.